### ISSUE TITLE
Register initial MarinDNA models and datasets

### ISSUE THREAD

**Issue by gonzalobenegas** [Open-Athena/marin-dna#506]
Populate the development registry and Hugging Face Storage Bucket with a small representative set after the storage pilot succeeds.
Use closed issue #273 as source material for checkpoint selection.
Register checkpoints only when they are selected for evaluation or reuse; the registry does not need every training step.

Each exact selected checkpoint receives its own model ID and immutable URI.
The physical path preserves the original Marin run name and step:

```text
hf://buckets/open-athena/marin-dna/marin_checkpoints/<original-marin-run-name>/hf/step-<N>/
```

Each directory must be directly loadable after local staging and must bundle the tokenizer.
The model README records context window, training dataset IDs, run, step, and available descriptive metadata.
It does not require the previous GCS checkpoint path or a final-versus-intermediate label.

Register at least one complete untokenized training dataset with `num_examples` and `num_base_pairs`.
Marin's tokenized GCS cache remains outside the registry.

Register at least two evaluation-dataset views that demonstrate shared physical storage with distinct split or configuration selection.
Each view records one `split`, optional `configuration`, `usage`, and `num_examples`.
It does not record `num_base_pairs`.
If an evaluation dataset must be prepared or converted for this issue, keep that code on a permanent issue branch and store the artifact below `issues/<issue-number>/<artifact-slug>/`.
Merge only the accepted registry entry and general registry tooling to `main`.

Every artifact records one commit-pinned `generated_by` link and may record one GitHub issue link.
Ordinary Hugging Face Model and Dataset repositories are optional release mirrors and are not required by this issue.

Done when:

- At least one model, one training dataset, and two evaluation-dataset views are registered.
- All bucket destinations were unused before upload and reject later replacement.
- Checkpoint contents include model configuration, weights, and tokenizer files required by the existing loader.
- Training dataset statistics are generated from the stored untokenized artifact.
- Evaluation example counts are generated from each exact registered view.
- Cross-references and remote existence checks pass.
- A remote smoke test stages and loads the model and opens every registered dataset view.
- The dashboard configuration is used as input where it already contains model facts instead of creating contradictory copies.
- No issue #473 configuration, result, or artifact is included.
